### PR TITLE
Srsuddath 32552 facilities health connect number

### PR DESCRIPTION
**Pull request by srsuddath**
## Description
This PR adds the health connect phone number to any health facility result for facilities in VISN 8

## Original issue(s)
department-of-veterans-affairs/va.gov-team#32552

## Testing done
manual testing

## Screenshots
image
image

## Acceptance criteria
- [ ] phone number is displayed for VISN 8 facilities

### CODE REVIEW

File: src/applications/facility-locator/components/search-results-items/VaFacilityResult.jsx [hunk 1]
```diff
@@ -11,7 +11,12 @@ import LocationOperationStatus from './common/LocationOperationStatus';
 import LocationDistance from './common/LocationDistance';
 import Covid19Alert from './common/Covid19Alert';
 
-const VaFacilityResult = ({ location, query, index }) => {
+const VaFacilityResult = ({
+  location,
+  query,
+  index,
+  showHealthConnectNumber,
```
Comment (kelsonic) [inline src/applications/facility-locator/components/search-results-items/VaFacilityResult.jsx:18]: Can you add this to propTypes?

File: src/applications/facility-locator/components/search-results-items/common/LocationPhoneLink.jsx [hunk 2]
```diff
@@ -50,7 +50,12 @@ export const renderPhoneNumber = (
 // NOTE: Do not use this component to display Covid19 appointment phone numbers.
 // Use Covid19PhoneLink instead.
 
-const LocationPhoneLink = ({ location, from, query }) => {
+const LocationPhoneLink = ({
+  location,
+  from,
+  query,
+  showHealthConnectNumber = false,
```
Comment (kelsonic) [inline src/applications/facility-locator/components/search-results-items/common/LocationPhoneLink.jsx:57]: Can you add this to propTypes?

File: src/platform/utilities/environment/index.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ export default Object.freeze({
    * you should instead consider using the API helper function defined in
    * platform/utilities/api to fetch data.
    */
-  API_URL: environment.API_URL,
+  API_URL: 'https://staging-api.va.gov',
```
Comment (kelsonic) [inline src/platform/utilities/environment/index.js:59]: I don't think you meant to commit this 😂

**Review by kelsonic — APPROVED** [review]

File: src/platform/utilities/environment/index.js (hunk 3 above)
Comment (srsuddath) [reply src/platform/utilities/environment/index.js:59]: ruh roh

File: src/applications/facility-locator/components/search-results-items/common/LocationPhoneLink.jsx (hunk 2 above)
Comment (srsuddath) [reply src/applications/facility-locator/components/search-results-items/common/LocationPhoneLink.jsx:57]: good catch! thanks!

File: src/applications/facility-locator/components/search-results-items/VaFacilityResult.jsx (hunk 1 above)
Comment (srsuddath) [reply src/applications/facility-locator/components/search-results-items/VaFacilityResult.jsx:18]: fixed
fixed

**Review by oseasmoran73 — APPROVED** [review]
If linting warnings can be applied, that would be ideal!

Marking as approved to not block you

**Pull request merged**